app/controllers/idv/image_uploads_controller.rb

```diff
@@ -31,5 +32,18 @@ def image_upload_form
     def store_encrypted_images?
       IdentityConfig.store.encrypted_document_storage_enabled
     end
+
+    def liveness_checking_enabled?
+      # todo: use config item,  UI options and sp configuration(ial_context)
```

Comment:
Is there more to do for this todo?

Reply:
@night-jellyfish have to see what UI side changes are.

Reply:
Do we want to merge the comment and come back to it? Is there a ticket to come back to it? 

Or wait to merge until we have the todo accomplished?